webaccess/src/webaccess.cpp

```diff
@@ -96,8 +96,18 @@ WebAccess::~WebAccess()
 
 void WebAccess::slotHandleRequest(QHttpRequest *req, QHttpResponse *resp)
 {
-    if(m_auth && ! m_auth->authenticateRequest(req, resp))
-        return;
+    WebAccessUser user;
+
+    if(m_auth)
```

Comment:
Bug here!

If `m_auth` is null (when no `--web-passwords` is specified) there is no default user.
This may be fixed by having default constructor set `WebAccessUserLevel::NOT_PROVIDED`, I'm not sure now about implications of this - will check on Monday.